include/ftxui/screen/color.hpp

```diff
@@ -322,6 +323,20 @@ class Color {
   uint8_t blue_ = 0;
 };
 
+inline namespace literals {
```

Comment:
An [inline namespace](https://en.cppreference.com/w/cpp/language/namespace#Inline_namespaces) makes its members also visible in the enclosing namespace. Which in this case means that you can do `using namespace ftxui;` or `using namespace ftxui::literals;` depending on whether you want to make all ftxui symbols visible in the current scope or only the UDL operator. This is considered best practice and the standard library also follows this pattern (s. [chrono-literals](https://en.cppreference.com/w/cpp/symbol_index/chrono_literals), [string-literals](https://en.cppreference.com/w/cpp/symbol_index/string_literals), etc.)

Reply:
I just didn't wanted to stay dumb.
I think I figured out why someone would like it. I believe this can be useful if you want to `using namespace ftxui::litterals` and not `using namespace ftxui`.